src/com/facebook/buck/go/GoBuckConfig.java

```diff
@@ -155,8 +166,10 @@ private Tool getGoTool(
     }
 
     CommandTool.Builder builder = new CommandTool.Builder(new HashedFileTool(toolPath.get()));
-    for (String arg : getFlags(extraFlagsConfigKey)) {
-      builder.addArg(arg);
+    if (!extraFlagsConfigKey.isEmpty()) {
```

Comment:
Why is this extra check necessary? If `getFlags` doesn't return an empty list for an empty key maybe it should return an optional, or have a precondition check that the key is not empty.